src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -300,6 +306,26 @@ public void SetDatabaseChangeVector(DocumentsOperationContext context, Dictionar
             ChangeVectorUtils.WriteChangeVectorTo(context, changeVector, tree);
         }
 
+        public BlittableJsonReaderObject GetDatabasePeriodicExportStatus(DocumentsOperationContext context)
+        {
+            var tree = context.Transaction.InnerTransaction.ReadTree(PeriodicExportStatusSlice);
```

Comment:
IIRC we are keeping status e.g. for ETL in ConfigurationStorage. Maybe it should go there?